### ISSUE TITLE
SVG icon doesn't have name

### ISSUE THREAD

**Issue by shimsa** [elementor/elementor#10189]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

With social icon widget (for example), when link is filled and SVG icon is used, then name is not present and we not achieve accessibility rules.

## Steps to reproduce

Choose icon (Font Awesome without problem, unlike SVG icon) and fill link

![social-icon-widget](https://user-images.githubusercontent.com/7617350/72667265-31f63080-3a1a-11ea-8621-2aac5810b824.png)

Difference in HTML between Font Awesome icon

![font-awesome-icon](https://user-images.githubusercontent.com/7617350/72667267-3884a800-3a1a-11ea-8ef5-6b2e855e3abf.png)

and SVG icon

![svg-icon](https://user-images.githubusercontent.com/7617350/72667270-3cb0c580-3a1a-11ea-9ee9-1eb57ef4ee92.png)

Google Lighthouse result

![lighthouse-result](https://user-images.githubusercontent.com/7617350/72667177-20605900-3a19-11ea-8285-cdfaa4b2d309.png)

## How to fix it?
I think best solution will be use SVG image Alt text.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by KingYes** [comment]
Fixed by #10235
